### PR TITLE
HPCC-26589 Document Logging in Containerized Environments

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: g-pan <greg.panagiotatos@lexisnexis.com>
@JamesDeFabia  optionally review

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+    <para>There are multiple available log processing solutions. You could
+    choose to integrate HPCC Systems logging data with any of your existing
+    logging solutions, or to implement another one specifically for HPCC
+    Systems data. Starting with HPCC Systems version 8.4, we provide a
+    lightweight, yet complete log-processing solution for your convenience. As
+    stated there are several possible solutions, you should choose the option
+    that best meets your requirements. The following sections will look at two
+    possible solutions.</para>
+
+    <sect2 id="elastic4HPCC_HelmDependencies">
```
Comment (rpastrana) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml:79]: the section id doesn't seem to match the content

**Review by rpastrana — COMMENTED** [review]
@g-pan looks good. I did leave a couple of comments.
As discussed offline, there are a few entries where the text runs off the page.

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+      describes a local, minimal Elastic Stack instance for HPCC Systems
+      component log processing. Once successfully deployed, HPCC component
+      logs produced within the same namespace should be automatically indexed
+      on the Elastic Search end-point. Users can query those logs by issuing
+      Elastic Search RESTful API queries, or via the Kibana UI (after creating
+      a simple index pattern).</para>
+
+      <para>Out of the box, the Filebeat forwards the HPCC component log
+      entries to a generically named index: 'fielbeat'-&lt;VERSION&gt;-
+      &lt;DATE_STAMP&gt; and writes the log data into 'hpcc.log.' pre-fixed
```
Comment (rpastrana) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/ContainerLogging.xml:94]: and writes the log data into 'hpcc.log.*' pre-fixed fields.

**Review by rpastrana — COMMENTED** [review]
@g-pan found an issue, see inline comment

**Review by rpastrana — APPROVED** [review]
@g-pan looks good to me

**Review by JamesDeFabia — APPROVED** [review]
good from my POV

**Pull request merged**